WordPress/Classes/Services/AccountService.h

```diff
@@ -93,6 +93,31 @@ extern NSString *const WPAccountEmailAndDefaultBlogUpdatedNotification;
 - (void)requestAuthenticationLink:(NSString *)email success:(void (^)())success failure:(void (^)(NSError *error))failure;
 
 
+/**
+ Connect an account a social service via an ID token.
+
+ @param service The name of the social service.
+ @param token The service's OpenID Connect (JWT) ID token for the user.
+ @param success
+ @param failure
+ */
+- (void)connectToSocialService:(NSString *)service
```

Comment:
Could this be an enum or at least a typedef instead of a string?

Reply:
A string enum would be perfect but... objc. :/  Maybe a typedef constant tho.